test/e2e/run_networking_test.go

```diff
@@ -533,6 +533,8 @@ EXPOSE 2004-2005/tcp`, ALPINE)
 	})
 
 	It("podman run network expose host port 18082 to container port 8000 using slirp4netns port handler", func() {
+		// 2024-04 this won't work with our changes in #22207
+		Skip("slirp4netns cannot handle paths > 108")
```

Comment:
I suggest to just nuke it, not like someone will ever go back and enable it back. It is better covered in `podman run port forward range` in the system tests so we loose nothing really.

And really this test is useless, it tests nothing really other than that the port 18082 is already bound not the important bit, if connectivity to the container works.